src/cf-core/usage.md

```diff
@@ -706,7 +706,17 @@ Sets the element to `14px` (in `em`s) based on the text size passed as
 
 ## Base typography
 
-### Webfonts
+### Webfont mixins
+
+#### :warning: These mixins are deprecated and will be removed in CFv5 :warning:
+
+Our
+[`@font-face` syntax](https://github.com/cfpb/capital-framework/blob/canary/src/cf-typography/src/licensed-fonts.less)
```

Comment:
Should this link to the `master` branch instead of `canary`?

Reply:
Good catch. Fixed in d82c3a0401457a22301dfffb2dcaac037021e948